cvo_override.yaml

```diff
@@ -0,0 +1,38 @@
+- op: add
+  path: /spec/overrides
+  value:
+  - kind: Deployment
+    group: apps/v1
+    name: cluster-monitoring-operator
+    namespace: openshift-monitoring
+    unmanaged: true
+  - kind: Deployment
+    group: apps/v1
+    name: machine-config-operator
+    namespace: openshift-machine-config-operator
+    unmanaged: true
+  - kind: Deployment
+    group: apps/v1
+    name: etcd-quorum-guard
```

Comment:
If we 'unmanage'  `machine-config-operator`, do we also need to unmanage this one?

Reply:
@cfergeau yes some of the operators which are part of different operators managed by CVO, I think those operators which are part of same namespace.

Reply:
Are you saying `etcd-quorum-guard` is an operator of its own, and that it's not managed by `machine-config-operator`?

Reply:
@cfergeau it is managed by CVO also.